build.sbt

```diff
@@ -212,20 +212,20 @@ lazy val kamon = project
     moduleName      := "commons-kamon",
     coverageEnabled := false,
     libraryDependencies ++= Seq(
-      "net.bytebuddy" % "byte-buddy-agent"              % "1.9.14",
-      "io.kamon"      % "kanela-agent"                  % "1.0.0-RC4",
-      "io.kamon"      %% "kamon-status-page"            % "2.0.0-RC1",
-      "io.kamon"      %% "kamon-instrumentation-common" % "2.0.0-RC2",
-      "io.kamon"      %% "kamon-executors"              % "2.0.0-RC2",
-      "io.kamon"      %% "kamon-scala-future"           % "2.0.0-RC2",
-      "io.kamon"      %% "kamon-akka"                   % "2.0.0-RC3",
+      "net.bytebuddy" % "byte-buddy-agent"              % "1.9.16",
+      "io.kamon"      % "kanela-agent"                  % "1.0.0",
+      "io.kamon"      %% "kamon-status-page"            % "2.0.0",
```

Comment:
we should put the kamon version in a variable? it's repeated in a lot of dependecies?

Reply:
You're right. Added.